main.js

```diff
@@ -34,6 +34,7 @@ define(function (require, exports, module) {
   var nickname = null
   var knownDocs = {}
   var room
+  var remoteCarets = new Set()
```

Comment:
I don't use ES6 anywhere else. Please use an object set and use keys to iterate.